source/features/show-names.tsx

```diff
@@ -56,7 +56,7 @@ async function init(): Promise<false | void> {
 				insertionPoint.after(
 					' (',
 					<bdo className="css-truncate">
-						<span className="css-truncate-target" style={{maxWidth: '200px'}}>
+						<span className="css-truncate-target v-align-middle" style={{maxWidth: '200px'}}>
```

Comment:
The right alignment for text is `baseline`

Reply:
That's what I thought too, but it's even worse. I tried every `vertical-align` value and `middle` is the ~best~ least worst compromise. We lose a few pixels of alignment in the comment headers and timeline events, but we get much better alignment in review headers.

We could also try to add some padding or a `transform: translate()`

Reply:
No, baseline is the correct one and it looks fine where I tried it. What's pulling the element out of the alignment is the avatar. Transform would just be tape to hold it together temporarily

Reply:
Baseline on review comment

<img width="212" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/111372352-26bf8480-8660-11eb-8ee8-169acd22fc77.png">

Reply:
Baseline on review request (zero change, it's the default)

<img width="356" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/111372498-52db0580-8660-11eb-8f9b-fd7f1f1d25dd.png">

Reply:
Baseline on regular comments (zero change, it's the default)

<img width="374" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/111372598-74d48800-8660-11eb-87be-700dc1dc6dd3.png">

Reply:
I see the issue on review comments

![image](https://user-images.githubusercontent.com/16872793/111373185-fd0f5900-8671-11eb-9fdf-80af643d1bf8.png)

Reply:
My screenshots are with `baseline` applied. I also see the issue on `main`

Reply:
> No, baseline is the correct one and it looks fine where I tried it.

On what element did you apply `vertical-align`? I can't get it to align properly :thinking:

Reply:
On this same element. However it works correctly on Safari, but in Chrome it doesn't, `overflow:hidden` throws it off. Investigating

Reply:
`display: inline-block` on `bdo` fixes it everywhere in every browser